src/main/java/menu/view/InputView.java

```diff
@@ -0,0 +1,40 @@
+package menu.view;
+
+import camp.nextstep.edu.missionutils.Console;
+import java.util.List;
+import menu.util.Parser;
+import menu.util.Spliter;
+import menu.util.Validator;
+
+public class InputView {
+    private static final String INPUT_COACH_MSG = "코치의 이름을 입력해 주세요. (, 로 구분)";
+    private static final String INPUT_INEDIBLE = "\n%s(이)가 못 먹는 메뉴를 입력해 주세요.\n";
+
+    public List<String> readCoach() {
+        while (true) {
+            try {
+                System.out.println(INPUT_COACH_MSG);
+                String input = Console.readLine();
+                String[] splitInput = Spliter.splitInput(input);
+                Validator.validateCoach(splitInput);
+                return Parser.parseInputToList(splitInput);
+            } catch (IllegalArgumentException e) {
+                System.out.println(e.getMessage());
+            }
+        }
```

Comment:
```suggestion
        try {
            System.out.println(INPUT_COACH_MSG);
            String input = Console.readLine();
            String[] splitInput = Spliter.splitInput(input);
            Validator.validateCoach(splitInput);
            return Parser.parseInputToList(splitInput);
        } catch (IllegalArgumentException e) {
            System.out.println(e.getMessage());
            return readCoach();
        }
```
`while(true)`를 사용하지 않고도 재입력 로직을 구현할 수 있습니다.

저는 보통 관련 로직을 컨트롤러에 작성하는 편이에요. view에 있는 입력 로직은 가능하면 1회만 가능하도록 구현하고, 컨트롤러에서 재입력 로직을 구현하려고 합니다. 왜냐하면 저는 `재입력`이라는 행위 자체도 비즈니스 로직이라고 생각하기 때문이에요.

Reply:
추가로 `,` 기호 옆에 공백이 입력되는 경우, 공백도 같이 이름에 입력이 되네요. trim을 해주는건 어떨까요?